tck/api/src/main/java/org/eclipse/microprofile/metrics/tck/metrics/GaugeTest.java

```diff
@@ -54,19 +54,40 @@ public static WebArchive createDeployment() {
 
     @Test
     public void testManualGauge() {
-        MetricID gaugeMetricID = new MetricID("tck.gaugetest.gaugemanual");
+        MetricID supplierGaugeMetricID = new MetricID("tck.gaugetest.supplierGaugeManual");
+        MetricID toDoubleFunctiongaugeMetricID = new MetricID("tck.gaugetest.toDoubleFunctionGaugeManual");
+        
+        Gauge<?> sGauge = metrics.getGauge(supplierGaugeMetricID);
+        
+        //Gauge that uses a ToDoubleFunction retrieves a Long value
+        Gauge<?> fGauge = metrics.getGauge(toDoubleFunctiongaugeMetricID);
+        
+        Assert.assertNull(sGauge);
+        Assert.assertNull(fGauge);
         
-        Gauge<?> gauge = metrics.getGauge(gaugeMetricID);
-        Assert.assertNull(gauge);
         gaugeMe();
-        gauge = metrics.getGauge(gaugeMetricID);
         
-        Assert.assertEquals(0, gauge.getValue());
-        Assert.assertEquals(1, gauge.getValue());
+        sGauge = metrics.getGauge(supplierGaugeMetricID);
+        fGauge = metrics.getGauge(toDoubleFunctiongaugeMetricID);
+        
+        Assert.assertEquals(0, sGauge.getValue());
+        //Gauge that uses a ToDoubleFunction retrieves a Long value
```

Comment:
as I mentioned in another comment, a gauge created from a `ToDoubleFunction` should return `Double` so the assertions on `fGauge` will also need to be updated to expect a Double